content/doc/book/blueocean/create-pipeline.adoc

```diff
@@ -0,0 +1,109 @@
+they can be added using the "Add" button.
+
+When done, click "Create Pipeline".
+Blue ocean will look at all branches for the selected repository,
+and will start a Pipeline run for each branch containing a `Jenkinsfile`.
+
+image:blue-ocean/create-pipeline/git-create-pipeline.png['Providing Git Repository Details']
+
+== Creating Pipelines for GitHub Repositories
+
+To create a Pipeline from a GitHub,
+start by selecting "GitHub" as the Source Control system.
+
+image:blue-ocean/create-pipeline/select-github.png['Creating Pipeline for Github Repositories']
+
+=== Provide a GitHub Access Token
+
+If this is the first time Pipeline Creation has been run,
+Blue Ocean will ask for a
+link:https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/[GitHub Access Token]
+to allow Blue Ocean to access your organizations and repositories.
+
+image:blue-ocean/create-pipeline/github-create-token.png['Add a Ocean GitHub Access Token']
+
+If you have not already created a access token, click on the link provided and
+Blue Ocean will navigate to the right page on GitHub,
+automatically selecting the appropriate permissions it will need.
+
+image:blue-ocean/create-pipeline/github-personal-access-token.png['Creating a Blue Ocean GitHub Access Token']
+
+=== Select a GitHub Account or Organization
+
+All repositories on Github are grouped by owner, either an account or organization.
+When creating Pipelines, Blue Ocean mirrors that structure,
+asking users to select an account or organization which owns the repositories
+from which it will add Pipelines.
+
+image:blue-ocean/create-pipeline/github-org.png['Selecting an Account or Organization']
+
+=== Single Repository or All Repositories
+
+Blue Ocean offers two styles of Pipeline creation, either "single Pipeline" or
+"discover all Pipelines".
+
+==== New Pipeline from a Single Repository
+
+Selecting "New Pipeline" allows the user select and create a Pipeline for
+a single Repository.
+
+image:blue-ocean/create-pipeline/github-new-pipeline.png['Creating a Single Pipeline']
+
+After selecting a repository Blue Ocean will scan all the branches in that Repository
+and will create a Pipeline for each branch containing a "Jenkinsfile" in the root folder.
+Blue Ocean will then run the Pipeline created for each branch in this process.
+
+If no branches in the selected repository have a "Jenkinsfile",
+Blue Ocean will offer to "Create a New Pipeline" for that repository,
+taking the user to the
+<<pipeline-editor, Pipeline Editor>> to create a Pipeline.
+
+==== Auto-discover Pipelines
+
+Selecting "Auto-discover Pipelines" scans all repositories belonging to the selected owner,
+and will create a Pipeline for each branch containing a "Jenkinsfile" in the root folder.
+
+image:blue-ocean/create-pipeline/github-auto-discover.png['Discover All Pipelines for Owner']
+
+This is a more advanced option, that should be used with care.
```

Comment:
That's a fair point, that "Advanced" is so subtle and gray that it obviously doesn't register to me. :smile: 

I think what I really dislike is the implication of "this should be used with care" which to _me_ makes it sound like this is a dangerous option to choose.

I infer from the context of the button that this option is Advanced not because it's dangerous, but because more sophisticated and advanced users are likely going to have multiple repositories available with a `Jenkinsfile`. If the paragraph reads less like a warning to newbs, then I'll be happy